src/main/java/org/springframework/data/redis/cache/RedisCacheManager.java

```diff
@@ -34,21 +34,20 @@
 /**
  * {@link CacheManager} implementation for Redis backed by {@link RedisCache}.
  * <p>
- * This {@link CacheManager} creates {@link Cache caches} on first write, by default. Empty {@link Cache caches}
- * are not visible in Redis due to how Redis represents empty data structures.
+ * This {@link CacheManager} creates {@link Cache caches} on first write, by default. Empty {@link Cache caches} are not
```

Comment:
This is not appropriate places to create lines breaks in (source) Javadoc, regardless of how it is rendered.

I spent quite a bit of effort to clean this sort of thing up.

Reply:
Javadoc formatting is automated, so we should not spend too much effort in manually aligning these as the formatter is going to apply its formatting anyways.

Reply:
I don't agree with this statement. When I am working on, and maintaining source code, I am looking at the Javadoc in the source, checking for accuracy, necessary edits, completeness, etc. I am not looking to the rendered (e.g. HTML) form.